wallet/src/Pos/Wallet/Web/Backup.hs

```diff
@@ -11,30 +11,48 @@ import           Universum
 
 import qualified Data.HashMap.Strict as HM
 import qualified Data.SemVer as V
-
 import           Pos.Crypto (EncryptedSecretKey)
+import           Pos.Crypto.Signing.Safe (emptyPassphrase, safeKeyGen)
+import           Pos.Wallet.Web.ClientTypes (AccountId (..), CAccountMeta (..),
+                                             CId, CWalletMeta (..), Wal)
 import           Pos.Util.Util (maybeThrow)
 import           Pos.Wallet.Web.Account (AccountMode, getSKById)
-import           Pos.Wallet.Web.ClientTypes (AccountId (..), CAccountMeta (..), CId,
-                                             CWalletMeta (..), Wal)
 import           Pos.Wallet.Web.Error (WalletError (..))
 import           Pos.Wallet.Web.State (getAccountMeta, getWalletMeta)
 import           Pos.Wallet.Web.Util (getWalletAccountIds)
+import           Test.QuickCheck (Arbitrary (..), elements)
 
 currentBackupFormatVersion :: V.Version
 currentBackupFormatVersion = V.initial & V.major .~ 1
 
-newtype WalletMetaBackup = WalletMetaBackup CWalletMeta
-newtype AccountMetaBackup = AccountMetaBackup CAccountMeta
+newtype WalletMetaBackup = WalletMetaBackup CWalletMeta deriving Show
+newtype AccountMetaBackup = AccountMetaBackup CAccountMeta deriving Show
 
 data WalletBackup = WalletBackup
     { wbSecretKey :: !EncryptedSecretKey
     , wbMeta      :: !WalletMetaBackup
     , wbAccounts  :: !(HashMap Int AccountMetaBackup)
-    }
+    } deriving Show
```

Comment:
@gromakovsky @pva701 I'm arriving late to the party, but are we sure that using the automatically-derived `Show` instance won't leak the `EncryptedSecretKey` in the logs or somewhere else?

Reply:
Yes, because we have
```
instance Show EncryptedSecretKey where
    show _ = "<encrypted key>"

instance B.Buildable EncryptedSecretKey where
    build _ = "<encrypted key>"
```

Reply:
Perfect, that was what I wanted to hear, thanks 😉